test/README

```diff
@@ -3,33 +3,49 @@
 This directory contains automated integration tests for Cockpit, and the support
 files for them.
 
-To run the tests on Fedora, you need to install the following packages:
-
-    $ sudo yum install npm qemu-kvm python \
-         firewalld libvirt-daemon libvirt-client libvirt-python \
-         openssl expect rsync curl xz
+To run the tests on Fedora, refer to the [HACKING](../HACKING.md) guide for
+installation of all of the necessary build and test dependencies. There's
+no need to trigger a build manually - the test suite preparation step below
+will handle that.
 
 You may need to reboot after installing the above packages. For initialization
 the following command needs to be run as root to setup the libvirt bridged network:
 
     $ sudo ./test/vm-prep
 
+Note that this step also attempts to ensure nested virtualization is enabled,
+and may emit a spurious `rmmod: ERROR: Module kvm_amd is not currently loaded`
+warning when running on an Intel CPU (or vice-versa).
```

Comment:
While I agree in principle, perhaps add it for now, and then take it out again when the script is fixed?

I didn't try to change the behaviour myself, as I'm not clear on the best way to go - the script has an apparently flawed assumption that "AMD/Intel KVM config file present -> system is running that kind of chip", but I don't know whether it should be addressed by finding a better query condition, or else by just ignoring the error when it happens.